github/copilot.go

```diff
@@ -53,6 +53,16 @@ type CopilotSeatDetails struct {
 	UpdatedAt               *Timestamp  `json:"updated_at,omitempty"`
 }
 
+// SelectedTeams represents the teams selected for Copilot for Business.
+type SelectedTeams struct {
+	SelectedTeams []string `json:"selected_teams"`
+}
+
+// SelectedUsers represents the users selected for Copilot for Business.
+type SelectedUsers struct {
+	SelectedUsers []string `json:"selected_users"`
+}
```

Comment:
Sorry, that's not what I meant.

The original feedback from @WillAbides was a good one - we shouldn't create these extra structs just for the requests.

Instead, they should be anonymous (I said "temporary" which is my fault) structs _WITHIN_ the functions where they are being used.  We use this trick many other places in this repo.

See my earlier example:

```go
body := struct {
  SelectedTeams []string `json:"selected_teams"`
}{
  SelectedTeams: teamNames,
}
req, err := s.client.NewRequest("POST", u, body)
```